### PR TITLE
Custom fields and clear logging

### PR DESCRIPTION
**Pull request by nihal223**
* Added logic to update custom fields UI
* Also uncommented updating the UI

### CODE REVIEW

File: controllers/getGpt3Triage.js [hunk 1]
```diff
@@ -25,23 +25,23 @@ function parseGpt3Response (choices) {
...
-  console.log(`customer request: ${chatText}`)
+  console.log(`\n\n customer request: ${chatText}`)
   const openai = openAiClient()
 
   const prompt = `The following is a support ticket from a customer, reply only with a JSON-formatted object like this:
     { "summary": "a summary of the ticket conversation", 
     "sentiment": "The sentiment the user is feeling narrowed to 1 or 2 words.",
     "priority": "low, normal, high, or urgent.",
-    "category": "feature request, subscription question, view game, find team, messaging, or other.", 
+    "category": "feature_request, subscription_question, view_game, find_team, messaging, or other.",
```
Comment (nihal223) [inline controllers/getGpt3Triage.js:35]: Following instructions from [here](https://developer.zendesk.com/documentation/ticketing/managing-tickets/creating-and-updating-tickets/#setting-custom-field-values)

> For fields with values defined by tags, such as drop-down lists, specify the option's tag name for the value property, not the option text displayed in the list. For example, if the option's display text is "HD 3000 color printer" and its tag is hd_3000, set the custom field's value as follows:

Comment (trevynmace-gc) [reply controllers/getGpt3Triage.js:35]: ahhh, that makes more sense for sure.

**Review by trevynmace-gc — APPROVED** [review]
This looks great! 🔥 :shipit:

**Comment by nihal223** [comment]
Sentiment worked on this ticket
https://gamechanger.zendesk.com/agent/tickets/974959

**Pull request merged**